lib/tasks/hours_ingest.rake

```diff
@@ -1,65 +0,0 @@
-# frozen_string_literal: true
```

Comment:
@cdoyle-temple Why is this file deleted in this PR?

Reply:
The file was removed from this branch as it should not have been there.

Reply:
Hours ingest is another pending PR.

Reply:
I had inadvertently pushed the hours refactor branch into this one and the removal of this file is part of the cleanup of that.